src/components/menu/item.tsx

```diff
@@ -55,6 +60,11 @@ const MenuItem: FC<Props> = ({
           <IconComponent width="24px" height="24px" />
         </span>
         {title}
+        {NotificationComponent ? (
+          <span className="w-12/12 text-right px-5">
+            <NotificationComponent width="14px" height="14px" />
```

Comment:
we have a bit of a mix between two patterns here...
if you'd like to have the notification icon in the components-pkg (which I think is a good idea), there is no point in passing a component. a `hasNotification` boolean prop would do the job. that's good to force a specific style, sizing, etc.

in contrast, if you expect the designer to want to customize the notification per project or per use-case, passing the component would be preferred. in that case, you could delete the Notification component in here

Reply:
oh, I realize you might have added it for the sake of storybook?

Reply:
I think the two cases are still valid to think about... will you ever want to render any other component there? or can that be the responsibility of the MenuItem that noone outside of here has to worry about?

Reply:
Thanks for the feedback @masone ! 
Yes, the Notification component is here for the sake of StoryBook. 
I got what you mean though.  I just wanted to leave the choice to return any component in case we want to use this NotificationComponent for something else in the future. For example we could still use it to return this red dot, and in addition, return a different icon for another menuItem. 
Right now there is no value in it and I agree that it would be better not to worry about which component to return outside of MenuItem and just go with a boolean ! 👍

Reply:
ok let's go for that then! makes using the component simpler in the project too!